packages/cli/src/init/element/element.ts

```diff
@@ -121,8 +121,14 @@ export function createElementGenerator(templateName: string):
   };
 
   class Polymer3ElementGenerator extends ElementGenerator {
-    // TODO(bicknellr): Why doesn't this inherit properly? Because `async` is
-    // compiled out?
+    // This is not a no-op: Yeoman only checks the object's prototype's own
```

Comment:
Let's add a mention of yeoman/generator#1065 here with these `not a no-op` comments.